src/app/util/af-types.h

```diff
@@ -143,9 +174,9 @@ typedef union
      */
     uint8_t * ptrToDefaultValue;
     /**
-     * Actual default value if the attribute size is 2 bytes or less.
+     * Default value of the attribute.
```

Comment:
Why this change to the comment?  The new comment does not seem to be correct if the attribute type is larger than 2-byte; in that case this field is _not_ really the default value in any sense; the `ptrToDefaultValue` points to where the default value is.